src/main/java/com/checkmarx/flow/service/IssueService.java

```diff
@@ -112,6 +116,8 @@ public IssueService(FlowProperties properties) {
                     String fileUrl;
                     ScanResults.XIssue currentIssue = xIssue.getValue();
 
+                    codeBashingService.addCodebashingUrlToIssue(currentIssue);
```

Comment:
Here we rely on codeBashingService internal state, as created by createLessonsMap. This should be fixed - please see my note above.